app/controllers/concerns/bot_actions.rb

```diff
@@ -53,6 +53,15 @@ def create # rubocop:disable Metrics/MethodLength
     end
   end
 
+  def new_destroy
```

Comment:
I feel like this should be named something like `destroy_confirmation` as this brings up the confirmation modal when we click on the `Delete` for a bot account. Thoughts @ChrisHuynh333  @joshsadam

Reply:
Agreed, that name definitely works better

Reply:
Thanks, my head got stuck in how rails has `new` for create dialogs, but agreed I like this better as well.
Changed in 44216f0f3d059fa28a6e3abbd5f78ff3419ceaf8